packages/settingeditor-extension/src/index.ts

```diff
@@ -42,15 +43,16 @@ namespace CommandIDs {
 /**
  * The default setting editor extension.
  */
-const plugin: JupyterLabPlugin<ISettingEditorTracker> = {
+const plugin: JupyterFrontEndPlugin<ISettingEditorTracker> = {
   id: '@jupyterlab/settingeditor-extension:plugin',
   requires: [
     ILayoutRestorer,
     ISettingRegistry,
     IEditorServices,
     IStateDB,
     IRenderMimeRegistry,
-    ICommandPalette
+    ICommandPalette,
+    ILabShell
```

Comment:
Should you have a labshell argument in your activate function below?

Reply:
(or delete that line here...)

Reply:
Thanks for spotting that! I went through all the places where the `ILabShell` was superfluous and removed it, but forgot to remove the dependency.